openapi/components/schemas/GatewayAccountConfig/BVNK.yaml

```
@@ -26,3 +26,13 @@ allOf:
           - merchantId
           - hawkAuthId
           - hawkAuthKey
+      settings:
+        type: object
+        description: BVNK settings object.
+        properties:
+          payoutCurrency:
+            type: string
+            description: Payout crypto currency.
```

I think correct cryptocurrency is one word.

cc: @noon-dawg

```suggestion
            description: Payout cryptocurrency.
```

---

Well spotted @nanobash!